packages/entity-server/src/routes/hierarchy/EntityDescendantsRoute.ts

```diff
@@ -19,10 +18,7 @@ export type DescendantsRequest = HierarchyRequest<
   HierarchyRequestBody,
   HierarchyRequestQuery & { includeRootEntity?: boolean }
 >;
-export class EntityDescendantsRoute extends Route<
-  DescendantsRequest,
-  HierarchyResponse<EntityResponseObject[]>
-> {
```

Comment:
Turns out that specifying the Response type was unnecessary (as it was inferred from the Request type anyway) so removing for simplicity 👍